compose/services.yml

```diff
@@ -162,3 +165,16 @@ services:
       BALENA_HAPROXY_KEY: ${OPENBALENA_ROOT_KEY}
       BALENA_ROOT_CA: ${OPENBALENA_ROOT_CA}
       HAPROXY_HOSTNAME: ${OPENBALENA_HOST_NAME}
+    volumes_from:
+      - cert-provider
+
+  cert-provider:
+    build: ../cert-provider
+    volumes:
+      - certs:/certs
+      - cert-provider:/usr/src/app/certs
+    environment:
+      ACTIVE: ${OPENBALENA_USE_ACME_CERT}
```

Comment:
Minor: I think we could rename this and the one below under a common prefix, like “OPENBALENA_ACME_CERT_ENABLED” and “OPENBALENA_ACME_CERT_PRODUCTION”.

Reply:
Agreed.